client-templates/nexus3/README.md

```diff
@@ -0,0 +1,21 @@
+# Nexus 3 accept.json
```

Comment:
Added in a `README` for this, or we can highlight this in the main repo README when we come to do it. Not sure where to put this info.